modules/storage-persistent-storage-nfs-selinux.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-nfs.adoc
+
+[id="nfs-selinux-{context}"]
+= SELinux
+
+By default, SELinux does not allow writing from a pod to a remote 
+NFS server. The NFS volume mounts correctly, but is read-only.
+
+To enable writing to a remote NFS server, follow the below procedure.
+
+.Prerequisites
+
+* The `docker-selinux` package must be installed. This package provides the
+`virt_use_nfs` SELinux boolean.
+
+.Procedure
+
+* Enable the `virt_use_nfs` boolean. The `-P` option makes this boolean
```

Comment:
Minor, but I would say "Enable the `virt_use_nfs` boolean using the following command." Just so the next sentence has context.

Reply:
Resolved.